Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_044.NpmCredentialsValidator.cs

```diff
@@ -101,7 +101,9 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NotFound ||
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NonAuthoritativeInformation)
                 {
-                    return ValidationState.NoMatch;
+                    message = $"The Uri {uri} is generic, no way to validate teh credential with for account {id}";
```

Comment:
>{uri} is generi [](http://example.com/codeflow?start=40&length=15)

I think this update is an overcorrection. If a uri returns NotFound we would still return the new message, which doesn't seem right to me.

Reply:
I update the message again.